Makefile

```diff
@@ -0,0 +1,45 @@
+.PHONEY: all binary
+
+BUILD_DIR=build_calico_kubernetes
+BUILD_FILES=$(BUILD_DIR)/Dockerfile $(BUILD_DIR)/requirements.txt
+
+default: all
+all: binary test
+binary: dist/calico_kubernetes
+test: ut
+
+kubernetesbuild.created: $(BUILD_FILES)
+	cd build_calico_kubernetes; docker build -t calico/kubernetes-build .
+	touch kubernetesbuild.created
+
+dist/calico_kubernetes: kubernetesbuild.created
+	# Build docker container
+	cd build_calico_kubernetes; docker build -t calico/kubernetes-build .
```

Comment:
You don't need to build the docker container here if you're depending on kubernetesbuild.created.